schemas/io-package.json

```diff
@@ -1179,21 +1179,67 @@
             ".+": {
               "anyOf": [
                 {
-                  "type": "string"
+                  "type": "string",
+                  "deprecationMessage": "Please use structure {link, color, pro, cloud, intro, icon, name, order} instead of string"
```

Comment:
This property is not in the specs. Either write `DEPRECATED:` in the description or simply remove this option so developers will get a warning from the IDE.

Reply:
Done